consensus/clique/ren.go

```diff
@@ -144,7 +144,7 @@ func (d *DNR) Watch(ctx context.Context, db ethdb.Database) {
 
 		lastBlock = lastBlockNumber.Uint64()
 
-		time.Sleep(time.Second * 30)
+		time.Sleep(5 * time.Second)
```

Comment:
might want to increase this or keep a configurable variable in the consensus config in genesis file?

Reply:
Yeah, let me revert this change, we can have a new PR to make this configurable.